jobs/v4beta1/conftest.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+from google.api_core.exceptions import NotFound
+
+import pytest
+
+import job_search_create_company
+import job_search_create_job
+import job_search_create_tenant
+import job_search_delete_company
+import job_search_delete_job
+import job_search_delete_tenant
+
+PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
+
+COMPANY_EXT_ID = "COMPANY_EXT_ID_{}".format(uuid.uuid4())
+POST_UNIQUE_ID = "TEST_POST_{}".format(uuid.uuid4())[:20]
```

Comment:
No action required.

As you know, `uuid.uuid4()` looks like this: "2cdb623e-8382-473b-851a-db4b8588d323". Since there are hyphens, it sometimes doesn't work for cloud resources if you slice it because some resource doesn't allow hyphen as the last character.

You can use `.hex` attribute:

```ipython
print("TEST_POST_{}".format(uuid.uuid4().hex)[:20])
>>> TEST_POST_0e30ec0082
```